src/families/tron/bridge/js.js

```diff
@@ -188,7 +273,35 @@ const getTransactionStatus = a =>
     })
   );
 
-const signAndBroadcast = () => never();
+const signAndBroadcast = (a, t, deviceId) =>
+  Observable.create(o => {
+    let cancelled = false;
+    const isCancelled = () => cancelled;
+    const onSigned = () => {
+      o.next({ type: "signed" });
+    };
+    const onOperationBroadcasted = operation => {
+      o.next({ type: "broadcasted", operation });
+    };
+    doSignAndBroadcast({
+      a,
+      t,
+      deviceId,
+      isCancelled,
+      onSigned,
+      onOperationBroadcasted
+    }).then(
+      () => {
+        o.complete();
+      },
+      e => {
+        o.error(e);
+      }
+    );
+    return () => {
+      cancelled = true;
+    };
+  });
 
 const prepareTransaction = async (a, t: Transaction): Promise<Transaction> =>
   Promise.resolve(t);
```

Comment:
we'll need to impl the prepareTransaction & getTransactionStatus to get full support (required for send form basically). we can do this later in second step.

Reply:
+1 better do that later !